common/scala/src/main/scala/whisk/http/ErrorResponse.scala

```diff
@@ -168,7 +168,7 @@ case class ErrorResponse(error: String, code: TransactionId)
 /** Custom rejection, wraps status code for response and a cause. */
 case class CustomRejection private (status: StatusCode, cause: String) extends Rejection
 
-object ErrorResponse extends Directives {
+object ErrorResponse extends Directives with DefaultJsonProtocol {
```

Comment:
Previously we marshaled the ErrorResponse case class ourselves with `                            resp.withEntity(marshalling.marshalUnsafe(ErrorResponse(msg.asString, transid)))`. This `marshalUnsafe` method no longer seems to exist in Akka. Now I am calling `.toJson` on the `ErrorResponse`. 

Ex: 
```
val error = ErrorResponse(ent.data.utf8String, transid).toJson
```